examples/pytorch_surrogate_model/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _examples-ml-surrogate:
+
+9 stage laser-plasma accelerator surrogate example
+==================================================
+
+Nine stages of laser-plasma accelerators.
+The laser-plasma accelerator elements are neural network surrogate models,
+previously trained and included in ``models``.
```

Comment:
Here is a good location to x-ref the WarpX workflow example once ready :)

Reply:
Sounds good, agreed

Reply:
The location will be:
```suggestion
previously trained and included in ``models`` (see the `full workflow with WarpX here <https://warpx.readthedocs.io/en/latest/usage/workflows/ml_dataset_training.html>`__).
```
after https://github.com/ECP-WarpX/WarpX/pull/4503 was merged.